test/E2ETests/SmokeTestsOnNanoServer.cs

```diff
@@ -263,6 +264,12 @@ public SmokeTestsOnNanoServer(ITestOutputHelper output, RemoteDeploymentConfig c
                     ApplicationBaseUriHint = applicationBaseUrl,
                     ApplicationType = applicationType
                 };
+
+                if (applicationType == ApplicationType.Standalone)
+                {
+                    deploymentParameters.AdditionalPublishParameters = "-r win10-x64";
```

Comment:
Might be good to add an assert, just in case this code is every copy+pasted into another place.